src/multitenant_sdk.js

```diff
@@ -63,21 +125,22 @@ const authApiSdkFns = (authApiEndpointInterceptors, ctx) => [
     path: 'token',
     fn: createAuthApiSdkFn({
       ctx,
-      interceptors: [
-        new FormatHttpResponse(),
-        ...authInterceptors,
-        ..._.get(authApiEndpointInterceptors, 'token'),
-      ],
+      interceptors: tokenInterceptors(authApiEndpointInterceptors),
     }),
   },
   {
     path: 'clientData',
```

Comment:
Do we need the `token` and `clientData` SDK fns? Won't the public API always be `clientAuthData`? Or did you mean that they might be useful during development?

Reply:
At the moment they're added for development and for testing the routes in isolation. We can remove them when everything else is wired together.